### PR TITLE
Add build image and launch from source in README

### PR DESCRIPTION
**Pull request by JinHai-CN**
### What problem does this PR solve?

Move the build image and launch from source back to README.

### Type of change

- [x] Documentation Update

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -208,6 +211,80 @@ Updates to the above configurations require a reboot of all containers to take e
 > $ docker-compose -f docker/docker-compose.yml up -d
 > ```
 
+## 🔧 Build light docker image
```
Comment (writinwaters) [inline README.md]:
```suggestion
## 🔧 Build a lite Docker image
```

File: README.md [hunk 2]
```diff
@@ -208,6 +211,80 @@ Updates to the above configurations require a reboot of all containers to take e
...
+
+This image is approximately 1 GB in size and relies on external LLM services, as it does not include document parsing and embedding models.
+
+```bash
+git clone https://github.com/infiniflow/ragflow.git
+cd ragflow/
+docker build -f Dockerfile.slim -t infiniflow/ragflow:dev-slim .
+```
+
+### 🔧 Build docker image
```
Comment (writinwaters) [inline README.md]: Why Build docker image falls under Build a lite Docker image?

File: README.md [hunk 3]
```diff
@@ -208,6 +211,80 @@ Updates to the above configurations require a reboot of all containers to take e
...
+
+This image's size is approximately 9 GB in size and can reference via either local CPU/GPU or an external LLM, as it includes document parsing, and embedding models.
```
Comment (writinwaters) [inline README.md]:
```suggestion
This image's size is approximately 9 GB in size and can reference via either local CPU/GPU or an external LLM, as it includes document parsing and embedding models.
```

**Review by writinwaters — CHANGES_REQUESTED** [review]

**Review by yuzhichang — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -208,6 +211,80 @@ Updates to the above configurations require a reboot of all containers to take e
...
+
+```bash
+pip3 install huggingface-hub
+python3 download_deps.py # Download document parsing and embedding models
+docker build -f Dockerfile -t infiniflow/ragflow:dev .
+```
+
+## 🔧 Launch service from source for development
+
+1. Install Poetry, if you already installed Poetry, please skip this step:
```
Comment (writinwaters) [inline README.md]:
```suggestion
1. Install Poetry, or skip this step if it is already installed:
```

File: README.md [hunk 5]
```diff
@@ -208,6 +211,80 @@ Updates to the above configurations require a reboot of all containers to take e
...
+   ```bash
+   curl -sSL https://install.python-poetry.org | python3 -
+   ```
+
+2. Get RAGFlow source code and install Python dependencies:
```
Comment (writinwaters) [inline README.md]:
```suggestion
2. Clone the source code and install Python dependencies:
```

File: README.md [hunk 6]
```diff
@@ -42,6 +42,9 @@
 - 🔎 [System Architecture](#-system-architecture)
 - 🎬 [Get Started](#-get-started)
 - 🔧 [Configurations](#-configurations)
+- 🔧 [Build light docker image](#-build-light-docker-image)
+- 🔧 [Build docker image](#-build-docker-image)
```
Comment (yingfeng) [inline README.md]: 🔨
🪚
🪛

Using these emoji instead

File: README.md [hunk 7]
```diff
@@ -208,6 +211,80 @@ Updates to the above configurations require a reboot of all containers to take e
...
+   export POETRY_VIRTUALENVS_CREATE=true POETRY_VIRTUALENVS_IN_PROJECT=true
+   ~/.local/bin/poetry install --sync --no-root # install RAGFlow dependent python modules
+   ```
+
+3. Launch the RAGFlow dependent services (MinIO, Elasticsearch, Redis, and MySQL) using Docker Compose:
+   ```bash
+   docker compose -f docker/docker-compose-base.yml up -d
+   ```
+
+   Add the following line to `/etc/hosts` to resolve all hosts specified in **docker/service_conf.yaml** to `127.0.0.1`:
```
Comment (writinwaters) [inline README.md]:
```suggestion
   Add the following line to `/etc/hosts` to resolve all hosts specified in **docker/service_conf.yaml** to `127.0.0.1`:  
```

File: README.md [hunk 8]
```diff
@@ -208,6 +211,80 @@ Updates to the above configurations require a reboot of all containers to take e
...
+   ```
+   127.0.0.1       es01 mysql minio redis
+   ```
```
Comment (writinwaters) [inline README.md]:
```suggestion
   ```  
```

File: README.md [hunk 9]
```diff
@@ -208,6 +211,80 @@ Updates to the above configurations require a reboot of all containers to take e
...
+   In **docker/service_conf.yaml**, update mysql port to `5455` and es port to `1200`, as specified in **docker/.env**.
```
Comment (writinwaters) [inline README.md:262]:
```suggestion
   In **docker/service_conf.yaml**, update mysql port to `5455` and es port to `1200`, as specified in **docker/.env**.  
```

File: README.md [hunk 10]
```diff
@@ -208,6 +211,80 @@ Updates to the above configurations require a reboot of all containers to take e
...
+
+4. If you cannot access HuggingFace, set the HF_ENDPOINT environment variable to use a mirror site:
```
Comment (writinwaters) [inline README.md]:
```suggestion
4. If you cannot access HuggingFace, set the `HF_ENDPOINT` environment variable to use a mirror site:
```

File: README.md [hunk 11]
```diff
@@ -208,6 +211,80 @@ Updates to the above configurations require a reboot of all containers to take e
...
+ 
+   ```bash
+   export HF_ENDPOINT=https://hf-mirror.com
+   ```
+
+5. Launch RAGFlow backend service:
```
Comment (writinwaters) [inline README.md]:
```suggestion
5. Launch the RAGFlow backend service:
```

File: README.md [hunk 12]
```diff
@@ -208,6 +211,80 @@ Updates to the above configurations require a reboot of all containers to take e
...
+   ```bash
+   source .venv/bin/activate
+   export PYTHONPATH=$(pwd)
+   bash docker/launch_backend_service.sh
+   ```
+
+6. Install frontend dependencies:
```
Comment (writinwaters) [inline README.md]:
```suggestion
6. Install frontend dependencies:  
```

**Review by yingfeng — APPROVED** [review]

File: README.md [hunk 13]
```diff
@@ -208,6 +211,80 @@ Updates to the above configurations require a reboot of all containers to take e
...
+   ```bash
+   cd web
+   npm install --force
+   ```
+7. Configure frontend to update `proxy.target` in **.umirc.ts** to `http://127.0.0.1:9380`:
+8. Launch RAGFlow frontend service:
```
Comment (writinwaters) [inline README.md]:
```suggestion
8. Launch the RAGFlow frontend service:  
```

File: README.md [hunk 14]
```diff
@@ -208,6 +211,84 @@ Updates to the above configurations require a reboot of all containers to take e
 > $ docker-compose -f docker/docker-compose.yml up -d
 > ```
 
+## 🪛 Build docker image without embedding model
```
Comment (writinwaters) [inline README.md]:
```suggestion
## 🪛 Build docker image without embedding models
```

**Review by writinwaters — CHANGES_REQUESTED** [review]

File: README.md [hunk 15]
```diff
@@ -208,6 +211,84 @@ Updates to the above configurations require a reboot of all containers to take e
...
+git clone https://github.com/infiniflow/ragflow.git
+cd ragflow/
+pip3 install huggingface-hub
+python3 download_deps.py # embedding models
+docker build -f Dockerfile.slim -t infiniflow/ragflow:dev-slim .
+```
+
+## 🪚 Build docker image with embedding model
+
+This image's size is approximately 9 GB in size and can reference via either local CPU/GPU or an external LLM, as it includes document parsing, and embedding models.
```
Comment (writinwaters) [inline README.md]:
```suggestion
This image's size is approximately 9 GB in size and can reference via either local CPU/GPU or an external LLM, as it includes document parsing and embedding models.
```

**Review by writinwaters — CHANGES_REQUESTED** [review]

File: README.md [hunk 16]
```diff
@@ -208,6 +211,84 @@ Updates to the above configurations require a reboot of all containers to take e
 > $ docker-compose -f docker/docker-compose.yml up -d
 > ```
 
+## 🪛 Build docker image without embedding model
+
+This image is approximately 1 GB in size and relies on external LLM and embedding services, as it only includes document parsing models.
```
Comment (writinwaters) [inline README.md]:
```suggestion
This image is approximately 1 GB in size and relies on external LLM and embedding services, as it includes document parsing models only.
```

**Review by writinwaters — CHANGES_REQUESTED** [review]

**Pull request merged**